osf/models/preprint_provider.py

```diff
@@ -40,13 +40,27 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     allow_submissions = models.BooleanField(default=True)
     additional_providers = fields.ArrayField(models.CharField(max_length=200), default=list, blank=True)
 
-    PREPRINT_WORD_CHOICES = (
-        ('preprint', 'Preprint'),
-        ('paper', 'Paper'),
-        ('thesis', 'Thesis'),
-        ('none', 'None')
+    PREPRINT = 'preprint'
+    PREPRINTS = 'preprints'
+    PAPER = 'paper'
+    PAPERS = 'papers'
+    THESIS = 'thesis'
+    THESES = 'theses'
+
+    PREPRINT_WORD_CHOICES_SINGULAR = (
+        (PREPRINT, 'Preprint'),
+        (PAPER, 'Paper'),
+        (THESIS, 'Thesis'),
     )
-    preprint_word = models.CharField(max_length=10, choices=PREPRINT_WORD_CHOICES, default='preprint')
```

Comment:
I'm pretty sure this breaks the API because you'd need to change the API serializer to look this up. But rather than fixing the API, let's just refactor how this works to something less disruptive.